src/alf/util/useColorModeTheme.ts

```diff
@@ -13,7 +13,11 @@ export function useColorModeTheme(): ThemeName {
   React.useLayoutEffect(() => {
     const theme = getThemeName(colorScheme, colorMode, darkTheme)
     updateDocument(theme)
-    SystemUI.setBackgroundColorAsync(getBackgroundColor(theme))
+
+    // must be default on iOS due to the formSheet modal presentation style
+    if (!isIOS) {
+      SystemUI.setBackgroundColorAsync(getBackgroundColor(theme))
+    }
```

Comment:
I forget where we landed on this, should check in a few places if this is problematic (off the top of my head I recall that modals were a problem. Might not be now though with our new ones 🤔). There's a few PRs we can go back and look at I think for context.

Worst case we can probably update this on the fly, but what you did here definitely might be a viable option.

Reply:
Oh, if we're going to use `fullscreen` I guess we can revert this part anyway? At least on the iPhone 14 it takes up the full screen so I don't see any borders.